frontend/src/lib/utils.tsx

```diff
@@ -1059,12 +1059,16 @@ export function identifierToHuman(identifier: string | number, caseType: 'senten
 }
 
 export function parseGithubRepoURL(url: string): Record<string, string> {
-    const match = url.match(/^https?:\/\/(?:www\.)?github\.com\/([A-Za-z0-9_.-]+)\/([A-Za-z0-9_.-]+)\/?$/)
+    const match = url.match(
+        /^https?:\/\/(?:www\.)?github\.com\/([A-Za-z0-9_.-]+)\/([A-Za-z0-9_.-]+)(\/(commit|tree|releases\/tag)\/([A-Za-z0-9_.\-\/]+))?/
```

Comment:
```suggestion
        /^https?:\/\/(?:www\.)?github\.com\/([A-Za-z0-9_.\-]+)\/([A-Za-z0-9_.\-]+)(\/(commit|tree|releases\/tag)\/([A-Za-z0-9_.\-\/]+))?/
```

I ran into this today when [building this](https://github.com/PostHog/posthog/pull/12476/files#diff-0b833302f2a06226982fc746d431ecbeef9f684c48e53b856cfcc4981a8c8d91).

Reply:
Might be worth double checking this elsewhere as well.

Reply:
I had a problem with the last part of that regex when it was like `[A-Za-z0-9_.-\/]+`. Turns out the `-` made a match range between `.` and `/` in this case. In the previous char group matches the `-` symbol is at the end of the group, and thus doesn't form a range, and probably doesn't need escaping... though it's probably good to do so anyway to avoid future bugs 🙈.

Reply:
Ahhhh good catch, I'll try and take a look through the others and see if anything else needs escaping